test/linter.js

```diff
@@ -116,15 +122,21 @@ describe('AutoRest Linter validation:', function () {
             return prFile.startsWith(configDir) && prFile.indexOf('examples') === -1 && prFile.endsWith('.json');
           }).forEach(prFileInConfigFile => {
             console.warn(`WARNING: Configuration file not found for file: ${prFileInConfigFile}, running validation rules against it in individual context.`);
-            execLinterCommand(`--input-file=${prFileInConfigFile}`);
+            errorsFound = errorsFound && execLinterCommand(`--input-file=${prFileInConfigFile}`);
```

Comment:
this will never execute since `errorsFound` is always false  right ?

Reply:
That makes sense. But surprisingly it was executing all 3 tags during my testing. hmm....

Just to be on the safe side, I have rearranged the order.